config/locales/server.en.yml

```diff
@@ -235,6 +235,10 @@ en:
     max_rows: "First %{max_bulk_invites} invites has been sent. Try splitting the file in smaller parts."
     error: "There was an error uploading that file. Please try again later."
 
+  invite_link:
+    email_taken: "This email is already in use. Please use different email address."
```

Comment:
This error message doesn't seem right to me because if an email is already in use, it means the user has signed up before. So they should either reset password or be redirected to the login page instead of being ask to use a different email.

Reply:
Updated the message to reflect they should login or reset password.